shared/metrics/src/main/java/io/pravega/shared/metrics/MetricsConfig.java

```diff
@@ -26,7 +26,7 @@
     public final static Property<Long> DYNAMIC_CACHE_SIZE = Property.named("dynamicCacheSize", 10000000L);
     public final static Property<Integer> DYNAMIC_CACHE_EVICTION_DURATION_MINUTES = Property.named("dynamicCacheEvictionDurationMs", 30);
     public final static Property<Integer> OUTPUT_FREQUENCY = Property.named("statsOutputFrequencySeconds", 60);
-    public final static Property<String> METRICS_PREFIX = Property.named("metricsPrefix", "pravega");
+    public final static Property<String> METRICS_PREFIX = Property.named("metricsPrefix", "localhost");
```

Comment:
I like the current approach a lot better. One minor point is that I don't fully get the reasoning behind the default value of `localhost`. It doesn't really have to be a host name or address, right? I'm specifically thinking about the case in which the application does not set it and there are multiple instances of the service. I don't necessarily think that `pravega` is a much better option, but at least it is general and doesn't try to imply anything else other than what service the instance is part of.

Reply:
The practical use of this setting is to provide an instance identifier, usually the hostname.   Honestly the config setting is too open-ended as it is (better: a `metrics.host` override).

Reply:
@fpj is `metrics.host` clearer for you than `localhost` to set the prefix default?

Reply:
I don't have any better suggestion, so I'm fine with it.

Reply:
Wow you guys seriously misunderstood my comment.   I meant that a config setting called 'metrics.host' would be better than the open-ended `metrics.prefix` (to future-proof Pravega for _dimensional_ rather than _hierarchical_ metrics).    Somehow you took this to mean that the default value should be `metrics.host`, which is completely nonsensical.  The two valid options are a) `localhost and b) the actual hostname.